### ISSUE TITLE
NumberFormatException upgrading to 1.0.1 

### ISSUE THREAD

**Issue by pabl0rg** [cbeust/kobalt#335]
`./kobaltw --update` fails with

```
Exception in thread "main" java.lang.NumberFormatException: multiple points
	at sun.misc.FloatingDecimal.readJavaFormatString(FloatingDecimal.java:1914)
	at sun.misc.FloatingDecimal.parseFloat(FloatingDecimal.java:122)
	at java.lang.Float.parseFloat(Float.java:451)
	at com.beust.kobalt.wrapper.Main.installDistribution(Main.java:170)
	at com.beust.kobalt.wrapper.Main.installAndLaunchMain(Main.java:80)
	at com.beust.kobalt.wrapper.Main.main(Main.java:17)
```

**Comment by cbeust** [comment]
Haha good one. Will fix.

**Comment by cbeust** [comment]
Should be fixed in 1.0.2, but you'll have to update manually, obviously.

I sent you a `kobalt-wrapper.jar` with the fix on Slack, you could also override your current with that one and try to update again.
